ui/components/DataTable.tsx

```diff
@@ -168,10 +168,7 @@ function UnstyledDataTable({
           <TableHead>
             <TableRow>
               {_.map(fields, (f) => (
-                <TableCell
-                  style={f.width && { maxWidth: f.width }}
-                  key={f.label}
-                >
+                <TableCell style={f.width && { width: f.width }} key={f.label}>
```

Comment:
I remember I had the widths as percentages originally - now that there's no max width on the table is that worth reconsidering?

Reply:
Yeah I think percentages would be easier to think about. I will change, although with `table-layout: fixed`, I have no idea what is going to happen.